src/components/ContentWrapper.jsx

```diff
@@ -330,8 +324,12 @@ const ContentWrapper = (props) => {
       </Menu.Item>
     );
   };
+
+  if (!user) return <></>;
+
   return (
     <>
+      {user?.attributes['custom:agreed_terms'] !== 'true' ? <PrivacyPolicyIntercept user={user} onOk={() => dispatch(loadUser())} /> : <></>}
```

Comment:
I spoke with a friend who is a front end engineer yesterday and he recommended to me to avoid using if statements that render `<></>` on false. He said that this is still adding more to the bundle. Instead, we should be using an &&, meaning we should write this like this:

```
{user?.attributes['custom:agreed_terms'] !== 'true' && (<PrivacyPolicyIntercept user={user} onOk={() => dispatch(loadUser())} />)}
```